lib/AppPluginZigbee/index.js

```diff
@@ -24,7 +24,14 @@ const NpmCommands = require('../Modules/NpmCommands');
 class AppPluginZigbee extends AppPlugin {
 
 	async run() {
```

Comment:
maybe replace the tabs with two spaces in this file?
Since git can filter whitespace changes they i think its usually fine to fix those issues when encountered